src/ResourceManager/Sql/Commands.Sql/help/Get-AzSqlInstanceDatabaseVulnerabilityAssessmentScanRecord.md

```diff
@@ -0,0 +1,224 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/az.sql/get-azsqlinstancedatabasevulnerabilityassessmentscanrecord
+schema: 2.0.0
+---
+
+# Get-AzSqlInstanceDatabaseVulnerabilityAssessmentScanRecord
+
+## SYNOPSIS
+Gets all vulnerability assessment scan record(s) associated with a given managed database.
+
+## SYNTAX
+
+### GetByNameParameterSet
+```
+Get-AzSqlInstanceDatabaseVulnerabilityAssessmentScanRecord [-InstanceName] <String> [-DatabaseName] <String>
+ [-ScanId <String>] [-ResourceGroupName] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### GetByResourceIdParameterSet
+```
+Get-AzSqlInstanceDatabaseVulnerabilityAssessmentScanRecord -InputObject <AzureSqlManagedDatabaseModel>
+ [-ScanId <String>] [-ResourceGroupName] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzSqlInstanceDatabaseVulnerabilityAssessmentScanRecord** cmdlet retrieves all vulnerability assessment scan record(s) associated with a given managed database.
+Note that you need to run *Enable-AzSqlInstanceAdvancedThreatProtection* and * Update-AzSqlInstanceDatabaseVulnerabilityAssessmentSettings* cmdlet as a prerequisite for using this cmdlets.
```

Comment:
Space between the `*` and `Update-Az...`

Reply:
Fixed